stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -638,14 +767,32 @@ public final class _TensorComputation {
 
   // NOTE: the following properties are intentionally not implemented as an enum
   // due to high churn, *please do not refactor for Swiftiness*.
+  private var stateTFE: TFEState?
   private var stateTF: TFState?
 
-  /// The thread to run tensor computation in. The global config flag
-  /// '_RuntimeConfig.usesSynchronousExecution' decides whether tensor
-  /// computation should be synchronous: if true, this property will always be
-  /// nil. Otherwise, this property is non-nil only when the tensor computation
-  /// is on-going.
-  private var pthread: pthread_t?
+  /// The threads to run tensor computation in. In eager mode, we use N threads
+  /// when the tensor computation involves N device functions. In non-eager
+  /// mode, we use a single thread (that's sufficient as these N device
+  /// functions can be sent to the same TF_SessionRun() call.)
+  ///
+  /// The global config flag '_RuntimeConfig.usesSynchronousExecution' decides
+  /// whether tensor computation should be synchronous: if true, this property
+  /// will always be nil. Otherwise, this property is non-nil only when the
```

Comment:
"this property will always be nil" -> "this property will always be empty". "Nil" refers exclusively to `Optional.none` in Swift.

Reply:
You are right. changed.